### ISSUE TITLE
按照官方教程出现异常 @sgwm-sutras/ui/style 的 style 文件 不存在

### ISSUE THREAD

**Issue by InfernalAzazel** [SharpGoldWaveMaker/sutras#1]
- 我已经按照 pnpm add -D @sgwm-sutras/plugin @sgwm-sutras/ui情况出现此问题

```
[plugin:vite:import-analysis]无法解析从“docs/.vitepress/theme/index.ts”导入“@sgwm-sutras/ui/style”。该文件是否存在？
/work/code/github/aa/docs/.vitepress/theme/index.ts:3:7
1 | 从“vitepress/theme”导入主题；
2 | 从“@sgwm-sutras/ui”导入{DemoBox}；
3 | 导入“@sgwm-sutras/ui/style”；
   | ^
4 | 从“naive-ui”导入naive；
5 | 导出默认{
```
![2024-06-04 11-59-50屏幕截图](https://github.com/SharpGoldWaveMaker/sutras/assets/40333954/1e188748-1552-4837-a576-6db17fcbbe9a)

以上图片可以看出依赖中并没有 style 文件

**Comment by SharpGoldWaveMaker** [comment]
@InfernalAzazel Thanks for your attention. I'll fix it as soon as possible

**Comment by InfernalAzazel** [comment]
> @InfernalAzazel Thanks for your attention. I'll fix it as soon as possible

Thank you for contributing to the Vue ecosystem. I have identified two issues.

1. Due to style separation in @sgwm sutras/ui 0.0.2
2. There is no intelligent prompt for sgwm @sutras/ui 0.0.2 under ts, so there should be no type processing done?

Their differences
@sgwm sutras/ui 0.0.1
![2024-06-04 13-31-51屏幕截图](https://github.com/SharpGoldWaveMaker/sutras/assets/40333954/81b6d72b-3e2d-4592-a297-699ae0475082)
@sgwm sutras/ui 0.0.2
![2024-06-04 13-34-22屏幕截图](https://github.com/SharpGoldWaveMaker/sutras/assets/40333954/d67355bb-f5f8-4213-a5c7-659d0adbb96b)

Currently, the rollback is running normally at 0.0.1.

![2024-06-04 13-24-01屏幕截图](https://github.com/SharpGoldWaveMaker/sutras/assets/40333954/e49491bc-2cf2-437e-a87e-db5b97b61bbd)

I'll try to see if I can use it now, because I used to use vitepress **theme demoblock**, but its demo box will break the style and become irregular due to 100vh

**Comment by SharpGoldWaveMaker** [comment]
> > @InfernalAzazel Thanks for your attention. I'll fix it as soon as possible
> 
> Thank you for contributing to the Vue ecosystem. I have identified two issues.
> 
> 1. Due to style separation in @sgwm-sutras/ui 0.0.2
> 2. There is no intelligent prompt for @sgwm-sutras/ui under ts, so there should be no type processing done?
> 
> Their differences @sgwm-sutras/ui 0.0.1 ![2024-06-04 13-31-51屏幕截图](https://private-user-images.githubusercontent.com/40333954/336317360-81b6d72b-3e2d-4592-a297-699ae0475082.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0ODk0OTcsIm5iZiI6MTcxNzQ4OTE5NywicGF0aCI6Ii80MDMzMzk1NC8zMzYzMTczNjAtODFiNmQ3MmItM2UyZC00NTkyLWEyOTctNjk5YWUwNDc1MDgyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDA4MTk1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxNDY3NzNiMDA4YTY5ZjVkNTU2ZTMyYmI2NTVkNTg3Mzg3OWZlZWMwMjZkMzE0ZDA3MzQ5YTNiM2U2OTBhNTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._GeLFQiddW_wduDoH-9LmLjtuqprMg4NEcMbHrrqE8s) @sgwm-sutras/ui 0.0.2 ![2024-06-04 13-34-22屏幕截图](https://private-user-images.githubusercontent.com/40333954/336317685-d67355bb-f5f8-4213-a5c7-659d0adbb96b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0ODk0OTcsIm5iZiI6MTcxNzQ4OTE5NywicGF0aCI6Ii80MDMzMzk1NC8zMzYzMTc2ODUtZDY3MzU1YmItZjVmOC00MjEzLWE1YzctNjU5ZDBhZGJiOTZiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDA4MTk1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThkNWRjZmJhNGQ2NjNhZTRhZDkyYTUwOWIzMTQ4MGU1YjUyMTVmZWNjNTkyYWNmMDc2MDE4YjdkNjJhZDNiMDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ujFX4eJMLl7zZJvWIrrGhHOhOgh2VCTsmISpHx_o1CE)
> 
> Currently, the rollback is running normally at 0.0.1.
> 
> ![2024-06-04 13-24-01屏幕截图](https://private-user-images.githubusercontent.com/40333954/336317998-e49491bc-2cf2-437e-a87e-db5b97b61bbd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0ODk0OTcsIm5iZiI6MTcxNzQ4OTE5NywicGF0aCI6Ii80MDMzMzk1NC8zMzYzMTc5OTgtZTQ5NDkxYmMtMmNmMi00MzdlLWE4N2UtZGI1Yjk3YjYxYmJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA0VDA4MTk1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxNzdlZmQ1NDM5N2NmM2I0ODAwMDUyMDhhNzA2YWVjMTY2Mzk3Y2Y5OTgxODI5ZmFkODFlZTc1YzI2NDA1YjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1uN20HUjxdYynf_cyBkqAE01UPiAyX0ZfVuCq8SCNvg)
> 
> I'll try to see if I can use it now, because I used to use vitepress **theme demoblock**, but its demo box will break the style and become irregular due to 100vh

I've released the new version @sgwm-sutras/ui@0.0.3

You may need to use sandbox mode to solve 100vh problems, see: https://sharpgoldwavemaker.github.io/sutras.docs/guide/preview/view#browser
